WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/FeeOrDiscountLineDetailsViewModel.swift

```diff
@@ -2,25 +2,28 @@ import SwiftUI
 import Yosemite
 import WooFoundation
 
-protocol FeeOrDiscountLineDetailsStringsProvider {
+protocol FeeOrDiscountLineTypeViewModel {
```

Comment:
Perhaps we can abstract the protocol name further, something like `PriceModifyingViewModelProtocol`. I have no good names 😅

Reply:
Adding the suffix `Protocol` doesn't seem necessary when we have more than one different instance (apart from the mock)... And PriceModifying doesn't seem to represent what the protocol is responsible for 😅 

I call it FeeOrDiscountLineTypeViewModel because we already have a type `LineType` inside `FeeOrDiscountLineDetailsViewModel`, and I add the `FeeOrDiscount` to make explicit where it belongs.